### PR TITLE
Use instance-number for IMU,BARO,MAG,GYR,ACC messages

### PR DESCRIPTION
**Pull request by peterbarker**
Tested in SITL.

I've also changed from log_Compass to log_MAG - we have a trend towards using the message name for these things.

### CODE REVIEW

**Comment by andyp1per** [comment]
FWIW APM Planner doesn't handle instance numbers, so this makes it essentially useless unless someone fixes it

**Comment by tridge** [comment]
@andyp1per we do have plot.ardupilot.org if we can't get an update to APM planner

File: libraries/AP_Logger/LogStructure.h [hunk 1]
```diff
@@ -1761,15 +1744,16 @@ struct PACKED log_PSC {
...
 // @Field: MagX: magnetic field strength in body frame
 // @Field: MagY: magnetic field strength in body frame
 // @Field: MagZ: magnetic field strength in body frame
-// @Field: OfsX: magnetic field offset in body frame
-// @Field: OfsY: magnetic field offset in body frame
-// @Field: OfsZ: magnetic field offset in body frame
+// @Field: OX: magnetic field offset in body frame
+// @Field: OY: magnetic field offset in body frame
+// @Field: OZ: magnetic field offset in body frame
 // @Field: MOfsX: motor interference magnetic field offset in body frame
```
Comment (rmackay9) [inline libraries/AP_Logger/LogStructure.h:1757]: I'd vote for shortening the MofsX fields intead of the OfsX fields

**Comment by rmackay9** [comment]
I very much like these "instance" fields because I find it saves me time when analysing a log using MP's dataflash log viewer.

**Review by tridge — APPROVED** [review]

**Pull request merged**